Source/Other/CategoryAdditions/SPMutableArrayAdditions.h

```diff
@@ -38,5 +38,7 @@
 @interface NSMutableArray (SPMutableArrayAdditions)
 
 - (void)reverse;
+- (nullable id)SPsafeObjectAtIndex:(NSUInteger)idx;
```

Comment:
let's remove the `SP` prefix for both and use just `safeObjectAtIndex` and `safeAddObject` :)

Reply:
done